proposals/PlayServicesVersion_Proposal.md

```diff
@@ -0,0 +1,58 @@
+# Play Services Version Proposal
+
+- Status: Proposed
+
+## Overview
+This is a proposal to offer an alternative solution for [CB-13145](https://issues.apache.org/jira/browse/CB-13145).
+
+## Problem
+Plugins using different versions of Google Play Services causes gradle to error out. Currently, the proposed solution is to create an Android only prefernce in `config.xml` to control the version of Google Play Services for the whole project. For instance, `config.xml` would look like this:
+`<preference name = "PlayServicesVersion" value = "11.0.0"/>`
+
+Then if, for example, the push plugin wanted to include FCM libraries in `plugin.xml`, it would contain:
+`<framework src="com.google.firebase:firebase-messaging:$PlayServicesVersion"/>`
+
+If the cordova-plugin-google-analytics wanted to include Play Services in its `plugin.xml` it would contain:
+`<framework src="com.google.android.gms:play-services-analytics:$PlayServicesVersion"/>`
+
+These plugins would both use the same version of Google Play Services. This gets around the problem of plugins using + in the framework tag and having two plugins pin different versions of Play Services.
+
+## Proposal
+This proposal recommends not using a key called PlayServicesVersion in `config.xml`. Instead, we want to use a generic way to define variables in `config.xml` that `plugin.xml` can use. The original solution assumes that both plugin authors are using the same PlayServicesVersion and expects the application author to add it to the `config.xml`.
+
+1) The plugin should auto add the variable it needs to `config.xml` if a default isn't there. So assuming we want `<preference name="PlayServicesVersion" value="11.0.1"/>` in `config.xml`, it would get auto added by plugin.xml if it doesn't already exist. If it does exist, don't touch it.
+
+2) We are still dependent on the plugin authors to use the standardized variables.
+
+3) We don't want `<preference name="PlayServicesVersion" value="11.0.1"/>`. It is too specific. 
+
+### Questions
+* Do we introduce a new tag to handle these types of cases? 
+* Do we augment on one of the many tags config.xml can currently have? [Config.xml reference](http://cordova.apache.org/docs/en/latest/config_ref/index.html)
+
+### How the variable tag works now
```

Comment:
this should come up soon after the propose heading. Probably doesn't need its own heading